build-tools/docker/Dockerfile

```diff
@@ -165,29 +165,18 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ####
 
 ####
-# OpenJDK 8
-####
+# OpenJDK 17
 # hadolint ignore=DL3008
-RUN apt-get -q update && apt-get -q install --no-install-recommends -y openjdk-8-jdk-headless \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y openjdk-17-jdk && \
```

Comment:
We might want to continue using `"&&"` at the beginning of lines.

Reply:
done. Fixed